.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,15 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+- If this is your first contribution, read our Getting Started guide https://github.com/kubernetes/community#your-first-contribution
+- If you are editing SIG information, please follow these instructions: https://github.com/kubernetes/community/tree/master/generator
+  You will need to follow these steps:
+  1. Edit sigs.yaml with your change 
+  2. Generate docs with `make all`. To build docs for one sig, run `make WHAT=sig-apps all`
+-->
+
+**What this PR does**:
+
+**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+
+**SIG changes**
+
+- [] If changing SIG information, I have read [these instructions](https://github.com/kubernetes/community/tree/master/generator) and re-generated the docs
```

Comment:
I don't think the stuff outside of the commented out section is needed. Community is more flexible, and many PRs here are proposals and such

Reply:
Yeah, good point. I'll remove this